libcec/Makefile

```diff
@@ -43,4 +43,18 @@ define Package/libcec/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib/
 endef
 
+
+define Package/cec-client
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=cec-client
+  DEPENDS:=+libcec
+endef
+
+define Package/cec-client/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) -L $(PKG_BUILD_DIR)/src/cec-client/cec-client $(1)/usr/bin/
```

Comment:
Do we need the -L?

Reply:
Yes, the `-L` is needed because apparently (I'm looking at my build_dir right now) the libcec build emits the binary with a version suffix, plus a softlink to it named `cec-client`. We want the actual binary in the package, not the softlink.